docs/design/two-phase-transfers.md

```diff
@@ -56,13 +52,30 @@ In the event that you post less than the amount you initially
 reserved, the rest of the amount not posted reverts back to the
 original account.
 
-## Timeouts
+### Void
+
+When the pending transfer is voided,
+[`debits_posted`](../reference/accounts.md#debits_posted) and
+[`credits_posted`](../reference/accounts.md#credits_posted) are not
+modified.
+
+### Timeout
 
 If a pending transfer is created with a timeout (which is optional),
```

Comment:
Link to the `timeout` field in the transfers reference.

Reply:
Does timeout as a feature exist yet btw?

Reply:
The field exists but currently does nothing :grin: 

(It depends on some internal parts of the query api. It will be part of our first release, though, so I think its fine to document it as if it exists.)

Reply:
Gotcha.